migrations/1528395579_.up.sql

```diff
@@ -0,0 +1,4 @@
+-- Adds an index on URI which is no queried more often. Not done in a
+-- transaction so we can create it concurrently.
+
+CREATE INDEX CONCURRENTLY IF NOT EXISTS uri_idx ON repo (uri);
```

Comment:
```suggestion
CREATE INDEX CONCURRENTLY IF NOT EXISTS repo_uri_idx ON repo (uri);
```

I think all indexes are prefixed with the table name.

Reply:
agreed. updated prod `ALTER INDEX uri_idx RENAME TO repo_uri_idx;`

Reply:
👍